### ISSUE TITLE
Tensorflow.NET chatbot label?? and Tensorflow.NET.Text

### ISSUE THREAD

**Issue by GeorgeS2019** [SciSharp/TensorFlow.NET#765]
I know SciSharp has chatbot repositories. e.g. BotSharp etc.
However, Tensorflow.NET repository has a **chatbot** label.

![image](https://user-images.githubusercontent.com/49812372/109126337-6e33ae80-774d-11eb-9fee-f16895088256.png)

Is there a plan to port tensorflow chatbot code (e.g. [transformer](https://blog.tensorflow.org/2019/05/transformer-chatbot-tutorial-with-tensorflow-2.html)) to tensorflow.NET?

Currently, it seems the [Tensorflow.NET.Text project ](https://github.com/SciSharp/TensorFlow.NET/projects/5) has not planned yet. 

Any feedback? 
On possible milestone towards Deep NLP using tensorflow.NET?
I also notice Seq2SeqSharp has been selected. Is there a plan to integrate Seq2SeqSharp to Tensorflow.NET or in whatever form?

**Comment by Oceania2018** [comment]
@GeorgeS2019 Currenty master branch is foucsed on make NLP/ NLU related task be runnable. The ultimate goal is to support BotSharp to use the state of art NLP model .NET.

**Comment by GeorgeS2019** [comment]
@Oceania2018  Do you see a path how [the NLU codes in Seq2SeqSharp](https://github.com/zhongkaifu/Seq2SeqSharp/issues/19), especially the transformer framework, could be useful in speeding up the NLP/NLU Keras implementation in Tensorflow.NET?
